pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
Can you put a card in our backlog to add more thorough tests?  This is a very small amount of testing for something that is going to be answering auth queries.